wazuh/wazuh-dashboard
# Change build number generation

### Description

This PR replaces the build number generation method to adhere to the previous Wazuh standard


### Issues Resolved

#283 

## Evidence

```console
[root@localhost vagrant]# jq -r '.build.number' /usr/share/wazuh-dashboard/package.json
49006
```

![image](https://github.com/user-attachments/assets/a7348eaf-9553-4c36-9f3c-3b53ad7a5b0b)


## Test
- Build a new package following the steps in [the documentation](https://github.com/wazuh/wazuh-documentation/blob/25794271b2ce0c14af81382c9e3c9087a557efec/source/development/packaging/generate-dashboard-package.rst)
- Install Wazuh using the generated package and verify the build number in the `package.json`
- In the browser dev-tools verify that the requests contains the build number

> [!TIP]
> I generated a custom [wazuh-install.zip](https://github.com/user-attachments/files/16799460/wazuh-install.zip) for 4.9.0 that uses a local `wazuh-dashboard.rpm` to install the dashboard. The `rpm` package should be in the same directory as the script, that can be executed with `bash wazuh-install.sh -a`. This only works for OSs that uses RPM.




## Check List
